### DISCUSSION TITLE
Scrollable issue

### DISCUSSION THREAD

**Discussion by SilvaLucas92** [Norserium/react-indiana-drag-scroll#78] [Q&A]
Hi how are you?I have a layout with a top and side bar with position fixed and another component where add items to make it scrollable, but i have an issue. when I reach the end of the component dragging, it moves the layout. How can i fix it? Im wrapping the whole component with the scrollbar.
Thanks!

**Comment by Norserium** [comment]
@SilvaLucas92, hello. I assume it happens on mobile device?

**Reply by SilvaLucas92** [reply]
No, in desktop. Should i use the useScroller hook?

**Reply by Norserium** [reply]
Could you record a video?
